examples/cra/src/components/app.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { App } from "./app";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+test("renders learn react link", () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```

Comment:
FYI for `toBeInTheDocument` you need to set up https://github.com/testing-library/jest-dom.

Reply:
Fixed and added a setupTests.ts